app/services/variant_units/option_value_namer.rb

```diff
@@ -56,7 +56,7 @@ def option_value_value_unit
     def option_value_value_unit_scaled
       unit_scale, unit_name = scale_for_unit_value
 
-      value = BigDecimal(@variant.unit_value / unit_scale, 6)
+      value = BigDecimal(@variant.unit_value / unit_scale, 6).truncate(2)
 
```

Comment:
Alternative solution would be to use `number_to_rounded` helper, but we would get 12.50 instead of 12.5

```ruby
::ActiveSupport::NumberHelper.number_to_rounded(value, precision: 2)
```

Reply:
Thanks Gaetan, it's great to point out the other options you considered.

From what I understand, truncating this number to maximum of two decimal places makes the most sense.